src/utils.ts

```diff
@@ -26,15 +26,9 @@ export function getPipePath(pipeName: string) {
     if (os.platform() === "win32") {
         return "\\\\.\\pipe\\" + pipeName;
     } else {
-        // On UNIX platforms the pipe will live under the temp path
-        // For details on how this path is computed, see the corefx
-        // source for System.IO.Pipes.PipeStream:
-        // tslint:disable-next-line:max-line-length
-        // https://github.com/dotnet/corefx/blob/d0dc5fc099946adc1035b34a8b1f6042eddb0c75/src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs#L340
-        return path.resolve(
-            os.tmpdir(),
-            ".dotnet", "corefx", "pipe",
-            pipeName);
+        // Windows uses NamedPipes where non-Windows platforms use Unix Domain Sockets.
+        // This requires connecting to the pipe file in different locations on Windows vs non-Windows.
```

Comment:
Technically the name difference is just .NET Core being strange, rather than the use of domain sockets, no?

Reply:
nah the location is different too. Named Pipes are in:

`\\.\pipe\foo`

domain sockets are in:

`/tmp/CoreFxPipe_foo`

Even the Named Pipes on Windows using .NET Standard instead are placed here:

`\\.\pipe\foo`